library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/Section.java

```diff
@@ -315,6 +315,15 @@ public void onBindFooterViewHolder(RecyclerView.ViewHolder holder) {
      * @return ViewHolder for the Item of this Section
      */
     public abstract RecyclerView.ViewHolder getItemViewHolder(View view);
+
+    /**
+     * Return the ViewType for a single item of this Section
+     * @param position position of the Item in the Section, not in the RecyclerView
+     * @return ViewType for the Item
+     */
+    public int getItemViewType(int position) {
```

Comment:
[DESIGN] I think the view type should be provided by `SectionRecyclerViewAdapter.getItemViewType` method. 
Depending on the design, SectionRecyclerViewAdapter could delegate it to Section but method should not be accessible by end user.